sdks/python/apache_beam/examples/wordcount_xlang_sql.py

```diff
@@ -0,0 +1,136 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""A cross-language word-counting workflow."""
+
+from __future__ import absolute_import
+
+import argparse
+import logging
+import re
+import typing
+
+from past.builtins import unicode
+
+import apache_beam as beam
+from apache_beam import coders
+from apache_beam.io import ReadFromText
+from apache_beam.io import WriteToText
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+from apache_beam.transforms.sql import SqlTransform
+from apache_beam.utils import subprocess_server
+
+MyRow = typing.NamedTuple([('word', unicode)])
+coders.registry.register_coder(MyRow, coders.RowCoder)
+
+
+class WordExtractingDoFn(beam.DoFn):
```

Comment:
Nuke this class and just do `FlatMap(lambda line: re.findall(r'[\w\']+', line))` or, since this is not the focus of this example, even simply `FlatMap(str.split)` and avoid the save main session ugliness.

Reply:
+1